plugins/Marketplace/vue/src/PluginList/CTAContainer.vue

```diff
@@ -34,7 +36,7 @@
         {{ translate('Marketplace_CannotUpdate') }}
         <span
           style="white-space:nowrap"
-        >(<HelpLink :plugin-name="plugin.name" />
+        >(<MoreDetailsAction @action="$emit('openDetailsModal')" v-if="!inModal" />
```

Comment:
Here, and behind the `Marketplace_CannotInstall` text, the `<span>` needs to be have the `!inModal` check. Otherwise the braces are displayed without contents.